### ISSUE TITLE
FileNotFoundError: [Errno 2] No such file or directory: '..\\VERSION'

### ISSUE THREAD

**Issue by balkon16** [google/sentencepiece#386]
Hi, 

I opened an [issue](https://github.com/huggingface/pytorch-transformers/issues/1090) relating to the `pytorch-transformers` library but was redirected here. For the sake of clarity here's all the relevant info:

OS: Windows10
Python: 3.5.2.
Error when trying `pip install sentencepiece`:
```
    ERROR: Command errored out with exit status 1:
     command: 'c:\users\pawel.lonca\appdata\local\programs\python\python35\python.exe' -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'C:\\Users\\PAWEL~1.LON\\AppData\\Local\\Temp\\pip-install-ibsvnyrj\\sentencepiece\\setup.py'"'"'; __file__='"'"'C:\\Users\\PAWEL~1.LON\\AppData\\Local\\Temp\\pip-install-ibsvnyrj\\sentencepiece\\setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base pip-egg-info
         cwd: C:\Users\PAWEL~1.LON\AppData\Local\Temp\pip-install-ibsvnyrj\sentencepiece\
    Complete output (7 lines):
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "C:\Users\PAWEL~1.LON\AppData\Local\Temp\pip-install-ibsvnyrj\sentencepiece\setup.py", line 29, in <module>
        with codecs.open(os.path.join('..', 'VERSION'), 'r', 'utf-8') as f:
      File "c:\users\pawel.lonca\appdata\local\programs\python\python35\lib\codecs.py", line 895, in open
        file = builtins.open(filename, mode, buffering)
    FileNotFoundError: [Errno 2] No such file or directory: '..\\VERSION'
    ----------------------------------------
ERROR: Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.
```

**Comment by taku910** [comment]
Windows whl packages for python 3.5 is not uploaded successfully because 3.5 doesn't accept the latest feature of distutils. Could you download the whl packages from the release pages and install it manually?

https://github.com/google/sentencepiece/releases

**Comment by balkon16** [comment]
@taku910, case closed. Thank you.

**Comment by balkon16** [comment]
It works. Case closed. Thank you.

**Comment by tkhan3** [comment]
OS : Redhat 7
Python :  3.7.6 (Anaconda)

I am getting below errors while installing it via pip:

```
(tanveer) [ai_u@powcbds ~]$ pip install sentencepiece
Collecting sentencepiece
  Using cached sentencepiece-0.1.83.tar.gz (497 kB)
    ERROR: Command errored out with exit status 1:
     command: /power8nfs/home/ai_u/.conda/envs/tanveer/bin/python -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-_4tb0alh/sentencepiece/setup.py'"'"'; __file__='"'"'/tmp/pip-install-_4tb0alh/sentencepiece/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base /tmp/pip-install-_4tb0alh/sentencepiece/pip-egg-info
         cwd: /tmp/pip-install-_4tb0alh/sentencepiece/
    Complete output (7 lines):
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-install-_4tb0alh/sentencepiece/setup.py", line 29, in <module>
        with codecs.open(os.path.join('..', 'VERSION'), 'r', 'utf-8') as f:
      File "/power8nfs/home/ai_u/.conda/envs/tanveer/lib/python3.7/codecs.py", line 904, in open
        file = builtins.open(filename, mode, buffering)
    FileNotFoundError: [Errno 2] No such file or directory: '../VERSION'
    ----------------------------------------
ERROR: Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.

```

**Comment by dilutedh2o** [comment]
@tkhan3 I'm having the same issue with python 3.7. Did you find a solution?

**Comment by ulissesferreira** [comment]
This is not closed. This still happens specially under Docker environments. Facing the same problem using Docker Python:3 image. Currently looking for a solution

**Comment by shawnohare** [comment]
I also have this issue in a fresh Python 3.8.2 virtualenv.

**Comment by songproducer** [comment]
I'm also having this issue after upgrading to python 3.8.2 Mac 10.15.4

**Comment by francisco-perez-sorrosal** [comment]
Same issue. Fresh conda env. python 3.8.2, Mac 10.15.2

**Comment by Kaspazza** [comment]
Same issue. python 3.8.2 macOS 10.15.4

**Comment by Kaspazza** [comment]
I tried different versions of python and it gives the same error on mac.

**Comment by alyaxey** [comment]
@balkon16 could you please reopen the ticket? It doesn't work for python 3.8.2, Mac 10.15.2

**Comment by balkon16** [comment]
Few people mentioned they have the same issue. Reopen.

**Comment by jerome9189** [comment]
same issue, Mac 10.15.4, python 3.8.2

**Comment by taku910** [comment]
Fixed on v0.1.86.
